### PR TITLE
Added node app

### PR DESCRIPTION
**Pull request by suptoasty**
Haven't started step 4 in the document yet.
DO NOT MERGE YET

### CODE REVIEW

**Comment by suptoasty** [comment]
@SimpsonDeveloper @nathanwoodruffoc , OK I think this one is good to merge? Try to set it up then run after that should be good to merge into master or dev.

**Review by nathanwoodruffoc — APPROVED** [review]
Looks good, runs on my machine

**Review by SimpsonDeveloper — APPROVED** [review]
It runs on my machine just fine. Seems good

**Pull request merged**
